test/x509aux.c

```diff
@@ -40,61 +40,59 @@ static int test_certs(int num)
 
     for (count = 0;
          !err && PEM_read_bio(fp, &name, &header, &data, &len);
-	 ++count) {
+         ++count) {
         int trusted = strcmp(name, PEM_STRING_X509_TRUSTED) == 0;
```

Comment:
Fair enough, I'll fix these.  I fix style problems when I notice them, I'm not looking hard.